Session/Shared/SessionTableViewController.swift

```diff
@@ -301,6 +301,8 @@ class SessionTableViewController<ViewModel>: BaseVC, UITableViewDataSource, UITa
             viewModel.updateTableData(updatedData)
             filteredTableData = filterTableDataIfNeeded()
             tableView.reloadData()
+            tableView.beginUpdates()
+            tableView.endUpdates()
```

Comment:
This seems odd - normally you would perform changes to the table between the `beginUpdates` and `endUpdates` calls, I'm wondering whether this would result in the table refreshing it's UI twice?

Reply:
Ugh that's a pain 😞 

No worries - in that case I think this is fine, the plan is to refactor this into SwiftUI anyway which _shouldn't_ have the same issue since it'd be proper data-driven UI